src/isar_turtlebot/robotinterface.py

```diff
@@ -49,67 +50,64 @@ def __init__(self):
         )
         self.current_task: Optional[str] = None
         self.inspection_status: Optional[TurtlebotStatus] = None
+        self.id: str = ""
```

Comment:
It is not obviously clear what the id attribute is referring to. Based on the code lower down it is the id of the current active task, but that is not clear based on the name id.